ptx/sec_differentials.ptx

```diff
@@ -159,20 +159,22 @@
     Clearly, <m>f(c) = \ell(c)</m>.
     Let <m>\dx</m> be a small number,
     representing a small change in the <m>x</m>-value.
-    We assert that:
+    We assert that
     <me>
       f(c+\dx) \approx \ell(c+\dx)
     </me>,
     since the tangent line to a function approximates well the values of that function near <m>x=c</m>.
-    This tangent line approximation is used frequently enough in applications that we give it a name.
+    In fact, this tangent line is the linear function that best approximates the value of <m>f(x)</m> for <m>x</m> near <m>c</m>. Because of its value in applications, we give it a name.
   </p>
 
   <definition xml:id="def-linearization">
     <statement>
       <p>
-        The function <m>\ell(x)</m> is often referred to as the <term>linearization</term>,
-        or <term>linear approximation</term> of <m>f</m> at <m>c</m>.
-        It is the linear function that best approximates the value of <m>f(x)</m> when <m>x</m> is close to <m>c</m>.
+        Let <m>f</m> be differentiable on an interval <m>I</m> containing <m>c</m>. The function <m>\ell(x) = \fp(c)(x-c)+f(c)</m> is the <term>linearization</term>,
+        or <term>linear approximation</term>, of <m>f</m> at <m>c</m>.
+        <!--
+				It is the linear function that best approximates the value of <m>f(x)</m> when <m>x</m> is close to <m>c</m>.
+				-->
         <idx><h>linearization</h></idx>
```

Comment:
By removing this line, aren't you deleting a reference to the only reason one would bother using the linearization? The whole point is not that it is _**a**_ linear approximation, but the _**best**_ linear approximation.

This could be moved out of the definition, perhaps, but doesn't it need to be mentioned?

Is not the *entire point* of differential calculus finding the best linear approximations to nonlinear functions?